app/public/components.css

```diff
@@ -23,6 +23,9 @@
   Colors - Brand Palette
 ..................................... */
 /* .....................................
```

Comment:
I'm noticing a lot of these comments without any CSS/SCSS in them, can we delete for now simply?

Reply:
It looks the build added comments I put in the color scss to all the css files. very strange. I'll see what I can do to reverse